app/views/categories/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<section class="category-section">
+  <div class="category-container">
+    <%= render "shared/head", title: 'categories', back: false %>
+    <div class="success-fail">
+      <% if flash[:notice] %>
+        <%= render 'shared/message', message: flash[:notice], status: 'success' %>
+      <% end %>
+
+      <% if flash[:alert] %>
+        <%= render 'shared/message', message: flash[:alert], status: 'danger' %>
+      <% end %>
+    </div>
+
+    <% if @categories.empty? %>
+      <div class="w-100  d-flex justify-content-center align-items-center">
+        <div class="getting-started-container w-100 p-3 p-md-5">
+          <h2 class="text-center">Getting Started?</h2>
+          <div class="introduction my-3 my-md-5">
+            <p>A mobile web application where you can manage your budget: you have a list of transactions associated with a category,
+              so that you can see how much money you spent and on what.</p>
+          </div>
+          <div class="for-category my-3 my-md-5">
+            <h2>Category</h2>
+            <p>
+              For each category, the user can see their name, icon and the total amount of all the transactions that belong to that category.
+              When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
+              There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
+            </p>
+          </div>
+          <div class="for-expense my-3 my-md-5">
+            <h2>Expense</h2>
+            <p>
+              Here, you can view and manage transactions in a specific category. The total category amount is displayed at the top. To add a new transaction, scroll down and click "Add a New Transaction." Use the "<" button to return to the home page. Enjoy!
+            </p>
+          </div>
+        </div>
+      </div>
+    <% else %>
+      <ul class="list-group mt-5 category-container bg-light">
+        <% @categories.each do |category| %>
+          <%= link_to category_expenses_path(category_id: category.id), class: 'list-group-item list-group-item-action d-flex justify-content-between align-items-center mb-3' do %>
+            <div class="text-4xl me-2 me-md-4"><%= category.icon %></div>
+            <div class="text-lg font-weight-bold"><%= category.name %></div>
+            <div class="text-lg font-weight-bold">$<%= format('%.2f', category.expenses.sum(:amount)) %></div>
+          <% end %>
+        <% end %>
+      </ul>
+    <% end %>
+
+    <div class="add-category-container d-flex justify-content-center px-3 px-md-5 py-3">
+      <%= link_to "ADD CATEGORY", new_category_path, class: 'btn btn-outline-light btn-lg' %>
+    </div>
```

Comment:
- [ ] Dear @kit0-0, I'm already impressed with your app, Although the layout is great I notice we can add something minor to follow up along side the provided template. I noticed your `add a new category` and `Add Epence` buttons are not fixed to the bottom of the screen. To follow the design guideline please make sure this button is stick to the bottom of the page.

### Your design 
<img width="422" alt="Screenshot 2023-10-25 at 10 00 55 at night" src="https://github.com/kit0-0/budget/assets/69077061/57d76397-195e-4da1-a061-cb8cc4d508d1">

### Original design 
<img width="422" alt="Screenshot 2023-10-25 at 10 10 17 at night" src="https://github.com/kit0-0/budget/assets/69077061/13e62b6d-972b-4675-b257-5592931c90ec">